examples/pennylane/7_Pulse_control_OQC_Lucy/7_Pulse_control_OQC_Lucy.ipynb

```diff
@@ -0,0 +1,753 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    jax.config.update("jax_enable_x64", True)
</pre><p>Are all the jax imports needed here? Keep in mind that most people will not be running this from a GPU instance as well. What can be simplified?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>The pulse module is restricted to just jax, so all the simulations in the notebook will require jax</p>